landoapi/transplants.py

```diff
@@ -871,14 +871,6 @@ def blocker_try_task_config(
     revision: dict, diff: dict, stack_state: StackAssessmentState
 ) -> Optional[str]:
     """Block revisions which contain the `try_task_config.json` file."""
-    if (
-        stack_state.landing_assessment
-        and stack_state.landing_assessment.landing_repo
-        and stack_state.landing_assessment.landing_repo.tree == "try"
-    ):
-        # `try_task_config.json` is allowed to be pushed to try.
-        return
-
```

Comment:
Is this being removed because we don't trigger landings to try via the UI? Will it be needed when we run checks on API-triggered landing jobs?

Reply:
Yes that's correct, these checks will only run on repos which exist on Phabricator. Once I add the API to run checks on API-triggered jobs I will add the check for `try` which skips the check.